tests/manage/pv_services/test_verify_rbd_based_rwo_pvc.py

```diff
@@ -12,50 +12,18 @@
     TimeoutExpiredError, CommandFailed, UnexpectedBehaviour
 )
 from tests import helpers
-from tests.fixtures import create_ceph_block_pool, create_rbd_secret
+from tests.fixtures import (create_ceph_block_pool, create_rbd_secret,
```

Comment:
please break the line after ( and put the rest on the separate line.  Just follow our conventions:
https://github.com/red-hat-storage/ocs-ci/blob/master/docs/coding_guidelines.md

Reply:
Fixed.